### PR TITLE
CB-8482 Burn images on spot (AWS, GCP)

### PR DESCRIPTION
**Pull request by Bajzathd**
Enable spot on AWS (had to diable ENA, but it is not a problem because the source ami used has it already enabled).
Enable preemptible on GCP.

### CODE REVIEW

File: packer.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "salt_path": "{{ env `SALT_PATH` }}",
     "pyzmq_version": "{{ env `PYZMQ_VERSION` }}",
     "aws_instance_type": "t3.2xlarge",
+    "aws_spot_price": "0.3328",
```
Comment (akanto) [inline packer.json:64]: Can we switch on spot without pricing?

Comment (Bajzathd) [reply packer.json:64]: According to the documentation we should be able to, but when I tried it tried to start a normal instance and could not find its type - this was the only working spot scenario I could find.

Comment (akanto) [reply packer.json:64]: Maybe we have an old version of packer?

Comment (Bajzathd) [reply packer.json:64]: Tried it with the newest version and the same happened - so I rolled back the version change

**Review by akanto — APPROVED** [review]

**Pull request merged**